src/javascript/app/Modules/SmartChart/Containers/smart-chart.jsx

```diff
@@ -47,6 +47,7 @@ class Chart extends React.Component {
                 chartStatusListener={this.props.getChartStatus}
                 chartType={this.props.chart_type}
                 endEpoch={this.props.end_epoch}
+                margin={this.props.margin ? this.props.margin : null}
```

Comment:
can't use `this.props.margin || null`?

Reply:
sure, I was following what the other params in this file are doing, will change those too then